tests/conftest.py

```diff
@@ -103,689 +86,958 @@ def get_settings_override() -> Settings:
-                    {"name": "country", "type": "STR"},
-                    {"name": "manager", "type": "INT"},
-                    {"name": "contractor_id", "type": "INT"},
-                ],
-            },
-        )
-        client.post(
-            "/nodes/contractors/table/",
-            json={
-                "database_name": "roads",
-                "catalog_name": "public",
-                "cost": 0.1,
-                "schema": "roads",
-                "table": "contractors",
-                "columns": [
-                    {"name": "contractor_id", "type": "INT"},
-                    {"name": "company_name", "type": "STR"},
-                    {"name": "contact_name", "type": "STR"},
-                    {"name": "contact_title", "type": "STR"},
-                    {"name": "address", "type": "STR"},
-                    {"name": "city", "type": "STR"},
-                    {"name": "state", "type": "STR"},
-                    {"name": "postal_code", "type": "STR"},
-                    {"name": "country", "type": "STR"},
-                    {"name": "phone", "type": "STR"},
-                ],
-            },
-        )
-        client.post(
-            (
-                "/nodes/repair_order_details/columns/repair_order_id/"
-                "?dimension=repair_order&dimension_column=repair_order_id"
-            ),
-        )
-        client.post(
-            (
-                "/nodes/repair_orders/columns/municipality_id/"
-                "?dimension=municipality_dim&dimension_column=municipality_id"
-            ),
-        )
-        client.post(
-            (
-                "/nodes/repair_type/columns/contractor_id/"
-                "?dimension=contractor&dimension_column=contractor_id"
-            ),
-        )
-        client.post(
-            (
-                "/nodes/repair_orders/columns/hard_hat_id/"
-                "?dimension=hard_hat&dimension_column=hard_hat_id"
-            ),
-        )
-        client.post(
-            (
-                "/nodes/repair_orders/columns/dispatcher_id/"
-                "?dimension=dispatcher&dimension_column=dispatcher_id"
-            ),
-        )
-        client.post(
-            (
-                "/nodes/local_hard_hats/columns/state_id/"
-                "?dimension=us_state&dimension_column=state_id"
-            ),
-        )
         yield client
 
     app.dependency_overrides.clear()
 
 
+def post_and_raise_if_error(client: TestClient, endpoint: str, json: dict):
+    """
+    Post the payload to the client and raise if there's an error
+    """
+    response = client.post(endpoint, json=json)
+    if not response.ok:
+        raise Exception(response.text)
+
+
+@pytest.fixture
+def load_examples():
+    """
+    Fixture function for loading all node examples to a client
+    """
+
+    def load_examples_fn(client: TestClient):
+        """
+        Load all node examples to a client
+        """
+        for endpoint, json in (
```

Comment:
YAML to JSON... yuck! :)

Reply:
Don't worry, the DJ python client will have the greatest YAML to JSON converter the world has ever known! 😂